src/components/auction/OngoingAuctionCard.tsx

```diff
@@ -0,0 +1,301 @@
+        requestType = RequestType.FillBadDebtAuction;
+        break;
+      case AuctionType.Liquidation:
+        requestType = RequestType.FillUserLiquidationAuction;
+        break;
+    }
+
+    const submitArgs: SubmitArgs = {
+      from: walletAddress,
+      to: walletAddress,
+      spender: walletAddress,
+      requests: [
+        {
+          amount: BigInt(100),
+          address: auction.user,
+          request_type: requestType,
+        },
+      ],
+    };
+
+    let response = await poolSubmit(pool.id, submitArgs, sim);
+    if (response && sim) {
+      setSimResponse(response);
+      if (SorobanRpc.Api.isSimulationSuccess(response)) {
+        setParsedSimResult(parseResult(response, PoolContract.parsers.submit));
+      } else {
+        console.error('Simulation failed', response);
+      }
+    }
+    setLoadingEstimate(false);
+    return response;
+  };
+  return (
+    <Section width={SectionSize.FULL} sx={{ flexDirection: 'column', marginBottom: '12px', ...sx }}>
+      <Box
+        sx={{
+          width: '100%',
+        }}
+      >
+        <Row>
+          <Typography variant="h3" sx={{ margin: '12px' }}>
+            Auction {toCompactAddress(auction.user)}
+          </Typography>
+          <Box
+            sx={{
+              margin: '6px',
+              padding: '6px',
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
+          >
+            <Box
+              sx={{
+                padding: '4px',
+                color: theme.palette.positive.main,
+                background: theme.palette.positive.opaque,
+                borderRadius: '5px',
+                lineHeight: '100%',
+              }}
+            >
+              *{AuctionType[auction.type]}*
+            </Box>
+          </Box>
+        </Row>
+        <Row>
+          <StackedTextBox
+            name="Started"
+            text={` ${auction.data.block} `}
+            sx={{ width: '50%' }}
+          ></StackedTextBox>
+          {
+            <StackedTextBox
+              name="Current Block"
+              text={`${currLedger}`}
+              sx={{ width: '50%' }}
+            ></StackedTextBox>
+          }
+        </Row>
+      </Box>
+      <LotList
+        pool={pool}
+        lot={scaledAuction.data.lot}
+        type={
+          auction.type === AuctionType.Interest || auction.type === AuctionType.BadDebt
+            ? 'Underlying'
+            : 'Collateral'
+        }
+      />
+      <DividerSection />
+      <BidList
+        pool={pool}
+        bid={scaledAuction.data.bid}
+        type={auction.type === AuctionType.Interest ? 'Underlying' : 'Liability'}
+      />
+
+      <OpaqueButton
+        palette={!parsedSimResult ? theme.palette.primary : theme.palette.positive}
+        sx={{ margin: '6px', padding: '6px' }}
+        onClick={() => handleSubmitTransaction(true)}
+      >
+        {!parsedSimResult ? 'Bid' : 'Update'}
```

Comment:
Can this say "Simulate Bid" instead of "bid"? if there is an issue in mobile cause the word is too long, "Sim Bid" is OK, but I would pref the former